src/server/ua_subscription.h

```diff
@@ -76,15 +66,31 @@ typedef struct UA_MonitoredItem {
     QueuedValueQueue queue;
 } UA_MonitoredItem;
 
+typedef struct MonitoredItem_queuedValue {
+    TAILQ_ENTRY(MonitoredItem_queuedValue) listEntry;
+    QueuedValueQueue *listHead;
```

Comment:
Getting the listhead is implicit in the `mon` pointer.

Reply:
Thanks ! Fixed.